src/common/ZSwitch.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Switch } from 'react-native';
+import { BRAND_COLOR } from './styles';
+
+export default class ZSwitch extends React.PureComponent {
```

Comment:
The only reason I called them otherwise is that React Native added a Button component recently, and the Switch one is a thin wrapper around the native one but adding a defaultValue functionality that the built-in didn't have for some reason.